MSVC-build.md

```diff
@@ -79,6 +76,6 @@ contents of the build log and open an issue on the tracker.
 Follow the common [instructions](https://gemrb.github.io/Install.html#configure-gemrb).
 
 If you want to run GemRB from within Visual Studio with several config files
-to be able to test several games at once, add new launcher configurations that
+to be able to test several games at once, add new [launcher configurations](https://docs.microsoft.com/en-us/cpp/build/configure-cmake-debugging-sessions?view=msvc-170) that
```

Comment:
Perhaps linking to the anchor `launchvsjson-reference` or mentioning it would be good, so it's easier to find where to specify arguments. But also, isn't this just about debug launches and if people try a normal run, it won't be any good?

Reply:
I found the beginning useful too, as someone who's never used cmake inside VS before, so I figured others would too.

As for normal launches, why would you go through all this trouble if that's what you want :P

Reply:
If you have to check something in the middle of debugging something else and you don't want to bother selectively disabling and reenabling breakpoints.

Reply:
Just checked, the same configuration is used for normal launch, you just run it with Ctrl+F5 instead of F5 :) ("Start without debugging")